clang/lib/CIR/Dialect/IR/CIRAttrs.cpp

```diff
@@ -0,0 +1,38 @@
+//===- CIRAttrs.cpp - MLIR CIR Attributes ---------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the attributes in the CIR dialect.
+//
+//===----------------------------------------------------------------------===//
+
+#include "clang/CIR/Dialect/IR/CIRDialect.h"
+
+using namespace mlir;
+using namespace mlir::cir;
```

Comment:
What about removing all the `using namespace` since this is a new start to the project?

Reply:
Those `using namespace` directives are in the Clang incubator project.  I would prefer to minimize the gratuitous differences between LLVM upstream and ClangIR incubator.

There is discussion about getting rid of the `::mlir::cir` namespace and moving everything therein to the `::cir` namespace.  If that happens, I will make the corresponding change upstream and will likely get rid of the `using namespace` directives then.